src/containers/Register/validations/passwordValidation.ts

```diff
@@ -0,0 +1,28 @@
+function isEmpty(str: string): boolean {
+  return str.length === 0;
+}
+
+function haveMinLength(password: string): boolean {
+  const passwordTrim = password.trim();
+  return passwordTrim.length >= 6;
+}
+
+function isInitBigLetter(password: string): boolean {
```

Comment:
I believe this shouldn't be a validation for a password, maybe just have a capital letter and not just the initial